lib/DDG/Spice/Tvmaze/Nextepisode.pm

```diff
@@ -9,18 +9,23 @@ spice is_cached => 1;
 spice proxy_cache_valid => "200 301 302 404 1h";
 
 triggers any => 'episode', 'airdate';
+triggers start => 'when does';
 
 spice to => 'http://api.tvmaze.com/singlesearch/shows?q=$1&embed=nextepisode';
 spice wrap_jsonp_callback => 1;
 
 handle query_lc => sub {
     return unless $_;
-
-    return $4 if $_ =~ /(next|upcoming) (episode|airdate) (in|of|for|from)? ?([a-z0-9 ]+)/;
-
-    return $2 if $_ =~ /(next|upcoming) ([a-z0-9 ]+) episode/;
-
-    return $1 if $_ =~ /([a-z0-9 ]+) (next|upcoming) (episode|airdate)/;
+        
+    return $4 if $_ =~ /(next|upcoming) (episode|airdate) (in|of|for|from)? ?(.+)/;
+        
+    return $2 if $_ =~ /(next|upcoming) (.+?) episode/;
+
+    return $1 if $_ =~ /(.+?) (next|upcoming) (episode|airdate)/;
+        
+    return $1 if $_ =~ /when does (.+?) (season [0-9]+) (start|come out|air)/;    
```

Comment:
would be good to add `premier` to the end:

![when_does_mr__selfridge_season_3_premiere_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15905214/14b9739a-2d82-11e6-85ec-03b817c0df0e.png)